.github/workflows/e2e-nvidia-l4-x1.yml

```diff
@@ -141,10 +141,6 @@ jobs:
           . venv/bin/activate
           ./scripts/e2e-ci.sh -m
 
-      - name: Check disk after tests
```

Comment:
Would it be ok to leave these?  They might come in handy later.

Reply:
Just for clarity, are you asking for both to stay? The new one added by Nathan that runs during the CI script, and this one that runs after at least some of the CI-generated artifacts have been cleaned up?

Reply:
Yes we can keep it for sure! Will update the patchset when I get in 👍

Reply:
Sounds reasonable - thanks for the explanation @danmcp :smile: